index.html

```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
+          
+        </div>
+        
+        </div>
+    </section>
+    <section>
+        <div class="container-1">
+            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (3).png" alt=""></a>
+           <div class="flow">
+            <h3 class="project-title-1">Multi-Post <br> Stories</h3>
+            <ul class="info">
+                <li class="conopy-li">
+                    <h3 class="canopy">CANOPY</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li class="back-end-li">
+                    <h3 class="back-end">Back End Dev</h3>
+                </li>
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
+                    <button class="button-1">html</button>
+                        
+                </li>
+                <li>
+                    <button class="button-2">css</button>
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
           
         </div>
-    
+        
+        </div>
+    </section>
+    <section>
+        <div class="about-my-self">
+        <div class="about-container">
+            <h1 class="about-title">About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2>LET’S CONNECT</h2>
```

Comment:
I suggest you change this to a link `<a>` or `<button>`tag. This should redirect users of your page to a contact form or to your social media handles.